File: 0164-maximum-gap/0164-maximum-gap.java
class Solution {
    public int maximumGap(int[] nums) {
        int n = nums.length;
        if(n<2){
            return 0;
        }
        Arrays.sort(nums);
        int dif = 0;
        for(int i=1;i<n;i++){
            dif = Math.max(dif,nums[i]-nums[i-1]);
        }
        return dif;
    }
}

File: 0164-maximum-gap/0164-maximum-gap.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var maximumGap = function(nums) {
    if(nums.length < 2) return 0;
    nums.sort((a,b) => a-b);
    let maxGap = 0;
    for(let i=1;i<nums.length;i++){
        maxGap = Math.max(maxGap, nums[i] - nums[i-1]);
    }
    return maxGap;
};